server-spi-private/src/main/java/org/keycloak/services/scheduled/ScheduledTaskRunner.java

```diff
@@ -50,26 +50,32 @@ public ScheduledTaskRunner(KeycloakSessionFactory sessionFactory, ScheduledTask
 
     @Override
     public void run() {
+        // trace a tracing provider directly to avoid creating a transaction that is unnecessary and would a surplus JTA transaction element to it
+        TracingProvider tracing = sessionFactory.getProviderFactory(TracingProvider.class).create(null);
```

Comment:
If there's no better approach for this, I'm ok with that.

Reply:
Thank you for the review, IMHO there isn't.